### DISCUSSION TITLE
Advertisement in the results (and no more files)

### DISCUSSION THREAD

**Discussion by c-wetzel** [openai/whisper#651] [General]
I just used the whisper command
`whisper <FILE PATH and Name>.mp4 --model large --language de --task transcribe`
with whisper 1.0 to transcribe a video and was quite amazed at the result. At the beginning and at the end of the transcription there are advertisements in the console. At the beginning "Untertitelung im Auftrag des ZDF, 2021" and at the end "Untertitel der Amara.org-Community" and several lines with "© Amara.org community".
This would not be a tragedy and can be easily deleted but contrary to my earlier attempts, whisper did not generate any files (*.txt, *.srt, *.vtt).

Does anyone know any advice here?

**Comment by arabcoders** [comment]
> with whisper 1.0 to transcribe a video and was quite amazed at the result. At the beginning and at the end of the transcription there are advertisements in the console. At the beginning "Untertitelung im Auftrag des ZDF, 2021" and at the end "Untertitel der Amara.org-Community" and several lines with "© Amara.org community".

This likely due to the training data, i noticed similar things with my Japanese's data set. it's not much though you can run the final result thru filter to remove the lines.

> This would not be a tragedy and can be easily deleted but contrary to my earlier attempts, whisper did not generate any files (*.txt, *.srt, *.vtt).

It should output normally, try setting the `--output_dir`  manually to writable directory.

**Reply by c-wetzel** [reply]
Thank you @ArabCoders,
for the tipp using the `--output_dir`. I have also tried the tiny and the medium model and got similar "spam" advertisement. Then I reinstalled whisper and now it seems to run just fine. So I am not sure if the data set is the culprit or if the ML went overboard.

**Reply by arabcoders** [reply]
I mean the training data itself might be tainted that's why you see some random spam sometimes =)

**Reply by bunthut** [reply]
Hi, I observed this in the chatgpt app when speech recognition is used. 

"Untertitel der Amara.org-Community"

Was added to my speech inputs several times

**Comment by livvius** [comment]
it seems only on cpu is happening not for mps
